docker/istio-dev/README.md

```diff
@@ -74,9 +74,35 @@ Check that you can access the cluster:
 kubectl get nodes
 ```
 
+To build istio and unit tests in docker container
+
+```bash
+cd ~/go/src/istio.io/istio
+```
+
+For istio building
+
+```bash
+sudo path=$PATH make BUILD_WITH_CONTAINER=0  build
+```
+
+For istio unit tests
+
+```bash
+sudo path=$PATH make BUILD_WITH_CONTAINER=0 test
+```
+
+some times when we want to rebuild it, you may find that istio cannot find the go path,
+one simple approach to solve it is to add go binary file to system bin/
+
+```bash
+cp /usr/local/go/bin/go /bin
+```
+
 ## Removing The Container
 
 ```bash
 docker stop istio-dev
 docker rm istio-dev
+make clean-dev-shell BUILD_WITH_CONTAINER=0
```

Comment:
Thank you for the hard work, however, this isn't the right approach. I would encourage you to merge whatever this container does into the build-tools container. the build-tools container is integrated into the common-files Make system.

Reply:
Hi sdake, thank you for your comment. the makefile clean-dev-shell command is to remove the created kubernetes clusters which is run by example's kind commnad. it is true that is should be separated from containers' theme